opensearch-cluster-cdk/README.md

```diff
@@ -0,0 +1,66 @@
+| singleNodeCluster (mandatory)                          | boolean  | Set `true` for single-node cluster else `false` for multi-node                           |
+| vpcId (Optional)                                       | string   | Re-use existing vpc, provide vpc id                                                      |
+| securityGroupId (Optional)                             | boolean  | Re-use existing security group, provide security group id                                |
+| cidr (Optional)                                        | string   | User provided CIDR block for new Vpc, default is `10.0.0.0/16`                           |
+| managerNodeCount (Optional)                            | number   | Number of cluster manager nodes, default is 3                                            |
+| dateNodeCount (Optional)                               | number   | Number of data nodes, default is 2                                                       |
+| clientNodeCount (Optional)                             | number   | Number of dedicated client nodes, default is 0                                           |
+| ingestNodeCount (Optional)                             | number   | Number of dedicated ingest nodes, default is 0                                           |
+
+#### Sample command to setup multi-node cluster with security disabled on x64 AL2 machine
+
+```
+cdk deploy "*" --context securityDisabled=true \
+--context minDistribution=false --context distributionUrl='https://artifacts.opensearch.org/releases/bundle/opensearch/2.3.0/opensearch-2.3.0-linux-x64.tar.gz' \
+--context cpuArch='x64' --context singleNodeCluster=false --context dataNodeCount=3 \
+--context dashboardsUrl='https://artifacts.opensearch.org/releases/bundle/opensearch-dashboards/2.3.0/opensearch-dashboards-2.3.0-linux-x64.tar.gz' \
+--context distVersion=2.3.0
+```
+
+### Interacting with OpenSearch cluster
+
+After CDK Stack deployment the user will be returned a load-balancer url which they can use to interact with the cluster.
+
+#### Sample commands
+`curl -X GET "http://<load-balancer-url>/_cluster/health?pretty"` for OpenSearch
+
+To interact with dashboards use port `8443`. Type `http://<load-balancer-url>:8443` in your browser.
+
+For security enabled cluster run `curl -X GET https://<load-balancer-url> -u 'admin:admin' --insecure`
+The security enabled dashboard is accessible using `http` on port `8443`
+
+#### Please note the load-balancer url is internet facing and can be accessed by anyone.
+To restrict access please refer [Client IP Preservation](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/load-balancer-target-groups.html#client-ip-preservation) to restrict access on internet-facing network load balancer.
+You need to add the ip/prefix-list rule in the security group created in network stack.
```

Comment:
Can we do something like https://github.com/opensearch-project/opensearch-ci/pull/171 to restrict server access?